packages/studio-web/src/app/soundswallower.service.spec.ts

```diff
@@ -583,12 +588,19 @@ AFwAWQBbAGEAagBvAHAAcABkAGIAaABhAGcAagBwAGkAVwBfAA==`;
 
 describe("SoundswallowerService", () => {
   let service: SoundswallowerService;
-  let audioService: AudioService;
-
+  let fileService: FileService;
+  let httpClientSpy: jasmine.SpyObj<HttpClient>;
+  let httpClient: HttpClient;
+  let httpTestingController: HttpTestingController;
   beforeEach(() => {
-    TestBed.configureTestingModule({});
+    httpClientSpy = jasmine.createSpyObj("HttpClient", ["get"]);
```

Comment:
Jest testing seems to work with nx and angular now... when I do the angular 17 migration I will eradicate Jasmine for good